app/common/src/WebView.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as React from 'react';
+import { WebView as NativeWebView } from 'react-native';
+
+import FullPageLoading from './loaders/FullPageLoading';
+
+type Props = {
+  // not exact - may contain additional properties (extend as needed)
+  source: {|
+    uri: string,
```

Comment:
Shouldn't the `source` have the same definition as in the native WebView?
https://facebook.github.io/react-native/docs/webview.html#source

Reply:
This is why "extend as needed". For me this is good for 2 things:

1) I don't have to spend additional time fixing it for the future.
2) We know what is actually used just by Flow type (I was actually considering making `source: string` because this is all we need now).